src/v2/WidgetRouter.js

```diff
@@ -12,15 +12,21 @@
 
 import FormController from './controllers/FormController';
 import BaseLoginRouter from './BaseLoginRouter';
+import DeviceEnrollmentTerminalViewController from './controllers/DeviceEnrollmentTerminalViewController';
 
 module.exports = BaseLoginRouter.extend({
   routes: {
     '': 'defaultAuth',
     '*wildcard': 'defaultAuth',
+    'authenticators/ov-not-installed': 'deviceEnrollmentTerminalView'
   },
 
   defaultAuth: function () {
     this.render(FormController);
   },
 
+  deviceEnrollmentTerminalView: function () {
+    this.render(DeviceEnrollmentTerminalViewController);
```

Comment:
pass options in render that set a flag. Meaning this controller does not contain any idx remediation responses and it is a terminal state.

`this.render(DeviceEnrollmentTerminalViewController, { terminalStateWithNoRemediation : true })`